atomic_reactor/plugins/exit_store_metadata_in_osv3.py

```diff
@@ -72,19 +87,26 @@ def get_repositories(self):
             "unique": unique_repositories,
         }
 
-    def get_digests(self):
+    def get_pullspecs(self, digests):
+        if self.workflow.push_conf.pulp_registries:
```

Comment:
Right. If someone needs this in the future we can make it smarter and match up the dockpulp environment name in config.